src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,18 @@
 package core.basesyntax;
 
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        int random = (int)Math.floor(Math.random() * COLORS.length);
+        return COLORS[random];
```

Comment:
do we need `Math.floor()`? how does casting to int work?

also, we don't really need to create a local variable here because we won't reuse it inside the method

Reply:
done